core/templates/domain/community_dashboard/FeaturedTranslationLanguageFactorySpec.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename the file to featured-translation-language.factory.spec.ts

Reply:
Done